build.ps1

```diff
@@ -0,0 +1,15 @@
+echo "Building the frontend..."
+cd frontend
+npm i
+npm run build
+cd ..
+echo "Copying frontend files..."
+mkdir target\web
+xcopy frontend\build target\web /s /e
+
+echo "Building the backend..."
+cd backend
+cargo build --release
+echo "Copying backend files..."
+mkdir target\hyde
```

Comment:
Same thing goes for all of this, please prefer native powershell commands over aliases or windows binaries